### PR TITLE
Add instance video details to sf-client output

### LINKED ISSUE #216: SF-Client does not show video details of instance

### PR DESCRIPTION
**Pull request by mandoonandy**
Resolves #216

### CODE REVIEW

File: shakenfist/client/main.py [hunk 1]
```diff
@@ -602,6 +603,7 @@ def _show_instance(ctx, i, include_snapshots=False):
     print(format_string % ('cpus', i['cpus']))
     print(format_string % ('memory', i['memory']))
     print(format_string % ('disk spec', i['disk_spec']))
+    print(format_string % ('video', i['video']))
```
Comment (mikalstill) [inline shakenfist/client/main.py:606]: video is a dict though, should we do a better job of formatting it here?

Comment (mandoonandy) [reply shakenfist/client/main.py:606]: The disk dumps a list of dict.
```
uuid        : c4d0bab2-6656-4a06-9f5f-0261c72e9f07
name        : bob
namespace   : system
cpus        : 1
memory      : 1024
disk spec   : [{'base': None, 'bus': None, 'size': 8, 'type': 'disk'}]
video       : {'memory': 16384, 'model': 'cirrus'}
node        : sf-3
power state : off
state       : created
console port: 48729
vdi port    : 38686

ssh key     : None
user data   : None

```
Change both or just video?
```
memory      : 1024
disk spec   : type:disk   bus:None  size:8   base:None
              type:disk   bus:ide   size:18  base:None
              type:cdrom  bus:ide   size:18  base:None
video       : model:cirrus  memory:16384
node        : sf-3
```
Defining characteristics of above:
* type is 5 char wide
* bus is min 4 wide
* size is min 2 char wide
* no space after colon
* two spaces after value

To make this more complicated I like moving the colon but I know everybody has a different taste.
```
uuid:          c4d0bab2-6656-4a06-9f5f-0261c72e9f07
name:          bob
namespace:     system
cpus:          1
memory:        1024
disk spec:     type:disk   bus:None  size:8   base:None
               type:disk   bus:ide   size:18  base:None
               type:cdrom  bus:ide   size:18  base:None
video:         model:cirrus  memory:16384
node:          sf-3
power state:   off
state:         created
console port:  48729
vdi port:      38686

ssh key:       None
user data:     None
```

Comment (mikalstill) [reply shakenfist/client/main.py:606]: I think this proposed format is much nicer that the current dumped JSON. I think we should change both. I think either format is fine as long as its neat, so you're choice.

**Comment by mandoonandy** [comment]
Output now looks like:
```
uuid        : 24cf11c1-e4f7-46bb-b3f5-b929aaa2c056
name        : bob3
namespace   : system
cpus        : 1
memory      : 1024
disk spec   : type=disk   bus=None  size=8   base=None  
              type=disk   bus=None  size=11  base=cirros  
              type=cdrom  bus=ide   size=12  base=cirros  
video       : model=cirrus  memory=16384  
node        : sf-3
power state : off
state       : created
console port: 43057
vdi port    : 32252
```

File: shakenfist/client/main.py [hunk 2]
```diff
@@ -563,6 +565,24 @@ def instance_list(ctx, all=False):
                          indent=4, sort_keys=True))
 
 
+def _pretty_data(row, space_rules):
+    ret = ''
+    for key in space_rules:
+        ret += key + '=' + str(row[key]).ljust(space_rules[key]) + '  '
```
Comment (mikalstill) [inline shakenfist/client/main.py:571]: TIL about ljust().

**Pull request merged**